## Description: Avoid mtt compilation in non linux OS, since it ends up using
##  linux/fb.h
## Author: Maximiliano Curia <maxy@debian.org>
Index: xawtv-3.102/x11/Subdir.mk
===================================================================
--- xawtv-3.102.orig/x11/Subdir.mk	2012-01-28 18:32:36.000000000 -0300
+++ xawtv-3.102/x11/Subdir.mk	2012-01-28 18:34:48.000000000 -0300
@@ -15,7 +15,7 @@
 TARGETS-x11 += \
 	x11/motv
 endif
-ifeq ($(FOUND_MOTIF)$(FOUND_ZVBI),yesyes)
+ifeq ($(FOUND_OS)$(FOUND_MOTIF)$(FOUND_ZVBI),linuxyesyes)
 TARGETS-x11 += \
 	x11/mtt
 endif
